my_uname/my_uname.c

```diff
@@ -17,3 +17,20 @@ int my_uname(struct utsname *buf) {
 
     return ret;
 }
+
+int main(int argc, char *argv[]) {
```

Comment:
functions shouldn't be named main() as it will cause conflict with the "real" main which serves as entry point. The function should not receive arguments now.

Reply:
Done